runhouse/servers/env_servlet.py

```diff
@@ -223,7 +223,7 @@ def _get_env_cpu_usage(self, cluster_config: dict = None):
         try:
             env_servlet_process = psutil.Process(pid=env_servlet_pid)
             memory_size_bytes = env_servlet_process.memory_full_info().uss
-            cpu_usage_percent = env_servlet_process.cpu_percent()
+            cpu_usage_percent = env_servlet_process.cpu_percent(interval=1)
```

Comment:
Will we want to increase these to 60 seconds and batch the calls so they don't wait for each env to finish?

Reply:
Actually... nevermind, probably best to check if this works first!